tests/modules/cnvkit/batch/test.yml

```diff
@@ -0,0 +1,101 @@
+- name: cnvkit batch test_cnvkit_tumoronly
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_tumoronly -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/reference.antitarget-tmp.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: output/cnvkit/reference.target-tmp.bed
+      md5sum: 26d25ff2d6c45b6d92169b3559c6acdb
+
+- name: cnvkit batch test_cnvkit_hybrid
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_hybrid -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/baits.antitarget.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
this is also emtpy md5sum, which sort of makes sense if the same bam files are compared with each other